### PR TITLE
Combine tests, reorganize debugging code

### PR DESCRIPTION
**Pull request by wthrowe**
### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

File: tests/Unit/Evolution/DgSubcell/Actions/Test_TciAndRollback.cpp [hunk 1]
```diff
@@ -286,8 +286,8 @@ void test_impl(const bool rdmp_fails, const bool tci_fails,
   using MockRuntimeSystem = ActionTesting::MockRuntimeSystem<metavars>;
   MockRuntimeSystem runner{{subcell_options}};
 
-  const TimeStepId time_step_id{true, self_starting ? -1 : 1,
-                                Time{Slab{1.0, 2.0}, {0, 10}}};
+  const TimeStepId time_step_id{false, self_starting ? -1 : 1,
```
Comment (kidder) [inline tests/Unit/Evolution/DgSubcell/Actions/Test_TciAndRollback.cpp:289]: I don't understand why this change is needed; what was not previously consistent?

Comment (wthrowe) [reply tests/Unit/Evolution/DgSubcell/Actions/Test_TciAndRollback.cpp:289]: Time was running backwards on lines 381 and 389, but forwards here.

Comment (kidder) [reply tests/Unit/Evolution/DgSubcell/Actions/Test_TciAndRollback.cpp:289]: okay, thanks

**Review by kidder — APPROVED** [review]
